scripts/run-migration.sh

```
@@ -0,0 +1,31 @@
+  echo "Error: Database configuration not found in .env file"
+  echo "Please ensure DB_HOST, DB_USER, DB_NAME, and DB_PASSWORD are set"
+  exit 1
+fi
+
+echo "Running migration against database: $DB_NAME on $DB_HOST"
+echo "User: $DB_USER"
+
+# Run the migration
+PGPASSWORD=$DB_PASSWORD psql -h $DB_HOST -U $DB_USER -d $DB_NAME -p ${DB_PORT:-5432} -f migration.sql
+
+if [ $? -eq 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)